README.adoc

```diff
@@ -1121,6 +1120,20 @@ users.where(id: ids)
 user.articles.order(:created_at)
 ----
 
+NOTE: When the receiver for `all` is an association, there are methods whose behavior changes by omitting `all`.
+
+The following methods behave differently without `all`:
```

Comment:
I think it might be better to leave the specifics of the differences to the official Rails documentation, rather than detailing them in this style guide.